.hlint.yaml

```diff
@@ -38,6 +38,7 @@
 - ignore: { name: 'Use const' }
 - ignore: { name: 'Move brackets to avoid $' }
 - ignore: { name: 'Use ?~' }
+- ignore: { name: 'Eta reduce' }
```

Comment:
Ignoring this one because it encourages losing names (`cancelCall = Grpc.serverCallCancel serverCall Grpc.StatusUnknown` instead of `cancelCall err = Grpc.serverCallCancel serverCall Grpc.StatusUnknown err`).

Reply:
Does this make it ignore project-wide? If so, I'd argue this warning is good to have in general. Is it possible to create a line/site-specific ignorance?